docs/reference/setup/restart-cluster.asciidoc

```diff
@@ -1,7 +1,7 @@
 [[restart-cluster]]
 == Full-cluster restart and rolling restart
 
-There may be {ref}/configuring-tls.html#tls-transport[situations where you want
+There may be {ref}/security-basic-setup.html[situations where you want
 to perform a full-cluster restart] or a rolling restart. In the case of
```

Comment:
Let's leave it for now — but this isn't great link text. As a user, I have no idea where the link is taking me. 
```suggestion
There may be <<security-basic-setup,situations where you want
to perform a full-cluster restart>> or a rolling restart. In the case of
```